xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-pageobjects/src/main/java/org/xwiki/ckeditor/test/po/MacroDialogEditModal.java

```diff
@@ -37,7 +37,8 @@ public class MacroDialogEditModal extends BaseElement
      */
     public MacroDialogEditModal waitUntilReady()
     {
-        getDriver().waitUntilElementIsVisible(By.cssSelector(".gadget-editor-modal .macro-name"));
+        getDriver().waitUntilElementIsVisible(
+                By.xpath("//*[contains(@class, 'editor-modal')]//*[contains(@class, 'macro-name')]"));
```

Comment:
Would be good to add a comment explaining why we use the suffix ``editor-modal`` instead of the full class name. And you can use a CSS selector, ``[attr*=value]``, see https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors .

    [class*='-editor-modal'] .macro-name